api-reference/beta/api/directoryrole_delete_member.md

```diff
@@ -0,0 +1,67 @@
+# Remove directory role member
+
+Use this API to create a new directory role member.
```

Comment:
Copy paste error by me!  Should say "Remove a member from a directoryRole.  Not sure if "directoryRole" should link to the resource description topic.